.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 
-dist: xenial
+dist: bionic
```

Comment:
Any particular reason for changing this? Bionic is already used for GitHub Actions, so better to keep testing on Xenial on Travis.

Reply:
I was trying to see what issues there were running on travis bionic.